wand/api.py

```diff
@@ -126,8 +126,48 @@ def load_library():
     raise IOError('cannot find library; tried paths: ' + repr(tried_paths))
 
 
```

Comment:
Artifacts left-over from a rebase/conflict-resolution? All of this has been moved to `wand.cdefs.structure` or `wand.cdefs.wandtypes`